fboss/platform/configs/montblanc/sensor_service.json

```diff
@@ -1,1894 +1,4180 @@
-      "3V3_L_VRM_VOUT": {
-        "compute": "@/1000",
-        "thresholds": {
-          "upperCriticalVal": 3.63,
-          "maxAlarmVal": 3.465,
-          "minAlarmVal": 3.135,
-          "lowerCriticalVal": 2.97
-        },
-        "path": "/run/devmap/sensors/3V3_L_MONITOR/in2_input",
-        "type": 1
-      },
-      "3V3_L_VRM_PIN": {
-        "compute": "@/1000000",
-        "thresholds": {
-          "upperCriticalVal": 887,
-          "maxAlarmVal": 760
-        },
-        "path": "/run/devmap/sensors/3V3_L_MONITOR/power1_input",
-        "type": 0
-      },
-      "3V3_L_VRM_POUT": {
-        "compute": "@/1000000",
-        "thresholds": {
-          "upperCriticalVal": 825,
-          "maxAlarmVal": 706.2
-        },
-        "path": "/run/devmap/sensors/3V3_L_MONITOR/power2_input",
-        "type": 0
-      },
-      "3V3_L_VRM_TEMP": {
-        "compute": "@/1000",
-        "thresholds": {},
-        "path": "/run/devmap/sensors/3V3_L_MONITOR/temp1_input",
-        "type": 3
-      }
-    },
-    "3V3_R_CARD": {
-      "3V3_R_U8_TEMP": {
-        "compute": "@/1000",
-        "thresholds": {
-          "upperCriticalVal": 85,
-          "maxAlarmVal": 80
-        },
-        "path": "/run/devmap/sensors/SMB_TSENSOR6/temp1_input",
-        "type": 3
-      },
-      "3V3_R_VRM_IOUT": {
-        "compute": "@/1000",
-        "thresholds": {
-          "upperCriticalVal": 250,
-          "maxAlarmVal": 214
-        },
-        "path": "/run/devmap/sensors/3V3_R_MONITOR/curr1_input",
-        "type": 2
-      },
-      "3V3_R_VRM_VIN": {
-        "compute": "@/1000",
-        "thresholds": {
-          "upperCriticalVal": 13.2,
-          "maxAlarmVal": 12.6,
-          "minAlarmVal": 11.4,
-          "lowerCriticalVal": 10.8
-        },
-        "path": "/run/devmap/sensors/3V3_R_MONITOR/in1_input",
-        "type": 1
-      },
-      "3V3_R_VRM_VOUT": {
-        "compute": "@/1000",
-        "thresholds": {
-          "upperCriticalVal": 3.63,
-          "maxAlarmVal": 3.465,
-          "minAlarmVal": 3.135,
-          "lowerCriticalVal": 2.97
-        },
-        "path": "/run/devmap/sensors/3V3_R_MONITOR/in2_input",
-        "type": 1
-      },
-      "3V3_R_VRM_PIN": {
-        "compute": "@/1000000",
-        "thresholds": {
-          "upperCriticalVal": 887,
-          "maxAlarmVal": 760
-        },
-        "path": "/run/devmap/sensors/3V3_R_MONITOR/power1_input",
-        "type": 0
-      },
-      "3V3_R_VRM_POUT": {
-        "compute": "@/1000000",
-        "thresholds": {
-          "upperCriticalVal": 825,
-          "maxAlarmVal": 706.2
-        },
-        "path": "/run/devmap/sensors/3V3_R_MONITOR/power2_input",
-        "type": 0
-      },
-      "3V3_R_VRM_TEMP": {
-        "compute": "@/1000",
-        "thresholds": {},
-        "path": "/run/devmap/sensors/3V3_R_MONITOR/temp1_input",
-        "type": 3
-      }
+    {
+      "slotPath": "/SMB_SLOT@0/OPTICR_SLOT@0",
+      "pmUnitName": "3_V_3_R_CARD",
```

Comment:
Why do we have two definitions for each version? Can we merge them into a single sensor list please?

E.g  for ("productProductionState": 2, "productVersion": 0,"productSubVersion": 10), 
one sensor list: [`3V3_L_VRM_IOUT`, `3V3_L_VRM_POUT`]
second sensor list: [`3V3_R_VRM_IOUT`, `3V3_R_VRM_POUT`]

Reply:
@kimdo8736 Do you mean the 3V3 card? There are two cards on the system, the left 3V3 card and the right 3V​​3 card, and they have different ID EEPROMs.

Reply:
Sorry, Looks like the right 3V3 card has ProductSubVersion 20 not 10.